src/components/MessageCard/MessageCard.jsx

```diff
@@ -81,6 +83,21 @@ export class MessageCard extends React.PureComponent {
     ) : null
   }
 
+  renderImage() {
+    const { image } = this.props
+
+    return image ? (
+      <ImageContainerUI>
+        <ImageUI
+          src={image.url}
+          alt={'Message image'}
```

Comment:
For accessibility reasons, maybe we can make allow for this to be passed, as `image.altText` or something like that and make the default 'Message Image' as you have it now.